tests/unit/amo/pages/TestSearchTools.js

```diff
@@ -20,4 +35,12 @@ describe(__filename, () => {
     );
     sinon.assert.callCount(fakeDispatch, 1);
   });
+
+  it('renders a NotFoundPage if server redirect fails', () => {
+    const { store } = dispatchClientMetadata();
```

Comment:
You don't need this line, nor the next line, and the render can just be `const root = render();`

Reply:
Good point! I had not considered that. I just updated the test.